homeassistant/components/fujitsu_hvac/const.py

```diff
@@ -0,0 +1,55 @@
+"""Constants for the Fujitsu HVAC (based on Ayla IOT) integration."""
+from ayla_iot_unofficial.fujitsu_hvac import FanSpeed, OpMode, SwingMode
+from bidict import bidict
```

Comment:
I think I know why you use bidict, but please just do it manually to avoid introducing a new dependency in the integration. If you still want to add it in the future, please try to add it in a follow up PR so there can be a separate discussion about this (As this is a use case more integrations have, so it makes sense to maybe apply to all places)